### PR TITLE
Update featured component to use top-nav component

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

Update featured component section on home page to use top-nav component.

### CODE REVIEW

File: src/fr/pages/index.md [hunk 1]
```diff
@@ -72,8 +72,8 @@ date: "git Last Modified"
...
-  <a href="{{ links.fieldset }}">En savoir plus sur le jeu de champs</a>
+  <h3 class="mb-400">Barre de navigation supérieure</h3>
+  <p class="mb-450">Une barre de navigation supérieure est une liste horizontale de liens de page. Utilisez une barre de navigation supérieure pour aider une personne à se repérer sur votre page Web ou site Web.</p>
+  <img class="d-block mb-450" src="../../images/common/components/preview-top-nav.svg" alt="UUn aperçu du composant barre de navigation supérieure qui montre la navigation du site représentée par des boîtes grises alignés horizontalement. Une boîte bleue suivi de deux boîtes grises représentent les liens où la dernière boîte est surlignée afin de représenter le lien actif." />
```
Comment (ethanWallace) [inline src/fr/pages/index.md:77]: Small typo - extra "U" at the start of the alt text

**Review by ethanWallace — CHANGES_REQUESTED** [review]
Just one small thing

File: src/fr/pages/index.md (hunk 1 above)
Comment (melaniebmn) [reply src/fr/pages/index.md]: Good catch!

**Review by ethanWallace — APPROVED** [review]
LGTM!

**Pull request merged**